docs/source/user-guide/install/index.rst

```diff
@@ -63,43 +69,50 @@ Follow the silent-mode instructions for your operating system:
 * :ref:`Linux <install-linux-silent>`
 
 
-Installing conda on a system that has other Python installations or packages
-============================================================================
+.. _hash-verification:
+
+Cryptographic hash verification
+===============================
+
+SHA-256 checksums are available for
+`Miniconda <https://docs.conda.io/projects/miniconda/en/latest>`_ and
+`Anaconda Distribution <https://docs.anaconda.com/free/anaconda/reference/hashes/all/>`_.
+We do not recommend using MD5 verification as SHA-256 is more secure.
+
+Download the installer file and before installing verify it as follows:
+
+* Windows:
+
+  * If you have PowerShell V4 or later:
+
+    Open a PowerShell console and verify the file as follows::
+
+      Get-FileHash filename -Algorithm SHA256
 
-You do not need to uninstall other Python installations or
-packages in order to use conda. Even if you already have a
-system Python, another Python installation from a source such as
-the macOS Homebrew package manager and globally installed
-packages from pip such as pandas and NumPy, you do not need to
-uninstall, remove, or change any of them before using conda.
+  * If you don't have PowerShell V4 or later:
 
-Install Anaconda or Miniconda normally, and let the installer
-add the conda installation of Python to your PATH environment
-variable. There is no need to set the PYTHONPATH environment
-variable.
+    Use the free `online verifier tool
+    <https://gallery.technet.microsoft.com/PowerShell-File-Checksum-e57dcd67>`_
+    on the Microsoft website.
 
-To see if the conda installation of Python is in your PATH
-variable:
+    #. Download the file and extract it.
 
-* On Windows, open an Anaconda Prompt and run ``echo %PATH%``
+    #. Open a Command Prompt window.
 
-* On macOS and Linux, open the terminal and run ``echo $PATH``
+    #. Navigate to the file.
 
-To see which Python installation is currently set as the default:
+    #. Run the following command::
 
-* On Windows, open an Anaconda Prompt and run ``where python``
+        Start-PsFCIV -Path C:\path\to\file.ext -HashAlgorithm SHA256 -Online
 
-* On macOS and Linux, open the terminal and run ``which python``
+* macOS: In iTerm or a terminal window enter ``shasum -a 256 filename``.
 
-To see which packages are installed in your current conda
-environment and their version numbers, in your terminal window
-or an Anaconda Prompt, run ``conda list``.
+* Linux: In a terminal window enter ``sha256sum filename``.
```

Comment:
Not critical, but you could also do all of the above with tabbing for each OS.

Reply:
We can do that in a future pull request.